client/src/containers/AddUser/AddUser.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { useState } from "react";
+import { InputField } from "../../components";
+import SmallContainer from "../../components/SmallContainer/SmallContainer";
+import callApi from "../../libs/axios";
+import { modules, methods } from "../../constants/constants";
+import { useAppContext } from "../App/AppContext";
+
+const AddUser = props => {
+  const [userEmail, setUserEmail] = useState(undefined);
+  const [userName, setUserName] = useState(undefined);
+  const { loginStatus } = useAppContext();
+
+  const handleUserEmailChange = e => {
+    setUserEmail(e.target.value);
+  };
+  const handleUserNameChange = e => {
+    setUserName(e.target.value);
+  };
+  const inviteUser = async () => {
+    const { post } = methods;
+    const { invitation } = modules;
+    const response = await callApi(post, `/${invitation}`, {
+      name: userName,
+      email: userEmail,
+      workspace: props.workspaceId,
+      type: 'WorkspaceInvite',
+      user: loginStatus.user._id,
+    });
+    console.log({response});
```

Comment:
can be removed

Reply:
We need user as we may have to send information to let the receiver know who sent the invite. We may need a type in the feature when there is a channel invitation. We need to distinguish what type of invitation is sent.